developers/contributing/governance.md

```diff
@@ -105,3 +105,16 @@ AC members can be excluded from the AC through an unanimous vote by the other AC
 When excluding a member, a reason has to be provided, such as inactivity, mis-behaviour, etc. and the excluded member must be given the chance to publicly respond to this.
 
 The openHAB Foundation's General Assembly can request that the AC is dissolved and has to be newly appointed.
+
+### Architecture Council Engagement
```

Comment:
Sleeping about this and seeing the actual effect of such a process, I think this goes against my original reason for establishing the AC, which is to remove bottlenecks and speed up decision making. A "government" does not need to do a people vote on every decision. This should at max be a case by case decision, if there's any doubt. So I'd suggest to leave not add this section as it effectively is not "a more detailed description", but rather a change in the process.